lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -191,3 +193,17 @@ def psfont(*args, **kwargs):
     ax.text(0.5, 0.5, 'hello')
     with tempfile.TemporaryFile() as tmpfile, pytest.raises(ValueError):
         fig.savefig(tmpfile, format='pdf')
+
+
+def test_pdf_savefig_when_color_is_none(tempdir_factory):
```

Comment:
Probably a typo, which is why it's not passing; should have been [`tmpdir_factory`](https://docs.pytest.org/en/latest/tmpdir.html#the-tmpdir-factory-fixture).